axelor-bank-payment/src/main/resources/views/BankReconciliationLine.xml

```diff
@@ -107,41 +107,52 @@
       onChange="action-bank-reconciliation-line-unreconcile-unselected-reconcile-selected"
       onSelect="action-bank-reconciliation-line-method-move-line-domain" form-view="move-line-form"
       grid-view="move-line-grid"/>
+    <field name="analyticDistributionTemplate"/>
```

Comment:
can you add a requiredIf="account.analyticDistributionRequiredOnMoveLines" and a readonlyIf="!account.analyticDistributionAuthorized" here? Everything is perfect after that